### ISSUE TITLE
Clang error when including file from other folder in stdafx.h

### ISSUE THREAD

**Issue by erikrenberg** [Caphyon/clang-power-tools#224]
I have a simple project and solution (attached at the end) showing a case where Clang Power Tools fails to compile the project because the stdafx.h file includes a folder in another folder. 

I am using the latest version of `clang-build.ps1` from the repo following the commit mentioned in https://github.com/Caphyon/clang-power-tools/issues/219

I get this error:
```
#include <fileinotherfolder.h>
// Error:
// 
// In file included from C : \Users\ErikR\source\repos\TestClangPowerTools\DllProject\DllProject.cpp:4 :
//   C : \Users\ErikR\source\repos\TestClangPowerTools\DllProject / stdafx.h:12 : 10 : fatal error : 'fileinotherfolder.h' file not found
```

The folder structure is as follows:

- **Sln** - contains the solution file and props file.
  - The props file uses two macros. The macro` $(UmLibsDir)` points to the Common sub folder.
- **DllProject** - contains the DLL project file, and cpp, h and stdafx-files.
  - The vcxproj file specifies that the props-file is to be used.
  - Specificies `AdditionalIncludeDirectories `like so:` $(UmLibsDir);%(AdditionalIncludeDirectories)`. This means that I can include files from that folder using this syntax: `#include <fileinotherfolder.h>`
- **Common**
  - Contains a single file `fileinotherfolder.h` that is included from the Dllproject CPP and .H-files.

This project builds fine with VC++, but fails with Clang Power Tools with the above error message.

Commenting out that line, and _including the same include from the DllProject.cpp or DllProject.h file works fine with Clang_. (Search the project for `#include <fileinotherfolder.h>` and you'll find the three places I have tested).

So there seems to be some issue with including header files in stdafx.h (that may or may not be related to issues #219 #209).

[TestClangPowerTools.zip](https://github.com/Caphyon/clang-power-tools/files/1688604/TestClangPowerTools.zip)

**Comment by cpp-red-lion** [comment]
Thank you for attaching detailed info and test project for this issue.
We'll look into it.

**Comment by erikrenberg** [comment]
Any update on this?

I have been checking other reported issues, and for a while I suspected that the cause of my issue is case-sensitivity of files and folders, as in issue #241, but I checked my sample project, and I think I include files with the same case as on the disk. So, probably not related.

**Comment by cpp-red-lion** [comment]
Hi @erikrenberg ,

Sorry for the delay.
It was an unusual _busy period_ for us (on other company projects).
And we don't yet have sufficient external contributors yet :(

I hope we'll get back on track with _Clang Power Tools_ backlog, as soon as next week.

Thank you for understanding.
Victor

**Comment by hero101111** [comment]
With the help of @dmateja this issue has been resolved (#282).
The next release of Clang Power Tools will include the fix.
